src/components/Expense/ExpenseChart.tsx

```diff
@@ -0,0 +1,28 @@
+import { getSpendingData } from '@/api/api'
+import { ExpenseDoughnutChart } from '@/components/Expense/ExpenseDoughnutChart'
+import { useQuery } from '@tanstack/react-query'
+
+const TODAY = new Date()
+const firstDay = new Date(TODAY.getFullYear(), TODAY.getMonth(), 1)
+const endDay = new Date(TODAY.getFullYear(), TODAY.getMonth(), TODAY.getDate())
+
+const ExpenseChart = (): JSX.Element => {
+	const { data: spendingData } = useQuery({
+		queryKey: ['spending', firstDay.toISOString(), endDay.toISOString()],
+		queryFn: () => getSpendingData(firstDay, endDay),
```

Comment:
쿼리키는 한국 시간대 및 포맷팅을 맞추기 위한 함수인 `dateToKorean` 을 가져와서 사용합시다! 현재는 TimeStamp까지 찍히고 있네요.
```suggestion
		queryKey: ['spending', dateToKorean(firstDay), dateToKoreean(endDay)],
		queryFn: () => getSpendingData(firstDay, endDay),
```

Reply:
콤마 있는게 좋아보여요, 수정해서 올릴게요